client/src/components/web-extension/Common/Grid/index.jsx

```diff
@@ -1,12 +1,56 @@
-import _get from 'lodash-es/get';
-import React from 'react';
+import React, { useCallback } from 'react';
 import PropTypes from 'prop-types';
-import { Bar } from '../../../Common/Spinner';
-import NoData from '../../../Common/NoData';
-import { getValueByFormat } from '../../../../helpers/format';
+import get from 'lodash-es/get';
+import isFunction from 'lodash-es/isFunction';
+import { getValueByFormat } from '@cd/helpers/format';
+import { Bar } from '@cd/common/Spinner';
+import NoData from '@cd/common/NoData';
 import './index.scss';
 
 const Grid = ({ data = [], metadata = {}, onRowClick, className, isLoading }) => {
+	const EMPTY_BALANCE = 0;
+	const getFormattedValue = useCallback((item, token) => {
+		const isEmptyBalance = Boolean(token?.balance?.displayValue === 0);
+		return item.key === 'price' && isEmptyBalance
+			? EMPTY_BALANCE
+			: getValueByFormat(item.value || get(token, item.key), {
+					format: item.format,
+			});
+	}, []);
+	const renderValue = useCallback(({ item, token }, value) => {
+		if (item.wrapperComponent) {
+			return <item.wrapperComponent>{value}</item.wrapperComponent>;
+		}
+
+		if (item.component) {
+			return <item.component {...(item.props && { ...item.props })} {...token} value={value} />;
+		}
+
+		return value;
+	}, []);
+	const renderGridItem = useCallback(
+		(item, token) => {
+			const formattedValue = getFormattedValue(item, token);
+			const shouldNotRender =
+				(isFunction(item?.shouldDisplay) && !item.shouldDisplay(get(token, item.key))) || !get(token, item.key);
+
```

Comment:
@duc-q The 0,03 is token price so it's expected to show there. 
The root cause is about this condition `!_get(value, item.key)`,  with 0 value it will return null.
So we need to update condition to `_get(value, item.key) === null` and keep the logic, no need to handle case by case for price or other value.

Reply:
expected result
![image](https://user-images.githubusercontent.com/6711653/184937364-95313296-e522-4e69-aa23-040e5cf85da7.png)

Reply:
Updated